_data/faq.yml

```diff
@@ -1254,7 +1254,72 @@
                 Es wird auf dem Bildschirm eine Schaltfläche angezeigt, die es dir erlaubt die Google Play Dienste zu installieren/aktualisieren. Dazu wird ein Google-Konto und Zugriff auf den Google Playstore benötigt.
                 
                 Wenn du dies nicht installieren kannst oder möchtest, kannst du stattdessen eine der OSM-Karten nutzen, die von c:geo unterstützt werden. Klicke hierzu einfach auf das Kartensymbol in der Titelzeile und wähle eine andere Karte anstatt Google aus.
+ 
+- id: ALC
+  title:
+    en: Adventure Lab Caches
+    de: Adventure Lab-Caches
+  items:
+      - id: ALC_PM
+        en:
+            title: Why are Adventure Lab Caches only shown to premium members?
+            content: |
+                There is no final information available from Groundspeak yet, how Adventure Labs will be implemented into their application interfaces and/or websites.
+                Therefore we decided for the time being to stick to a minimum approach, which is following the recommendation by Groundspeak and equals the implementation of other Geocaching apps.
```

Comment:
I would more explicitly move this decisions cause towards Groundspeak e.g.:

Therefore Groundspeak explicitly asked us to include this restriction, which is also present in other Geocaching apps. In spirit of good collaboration of c:geo and Groundspeak we decided to respect their wishes for now.

Reply:
Sound good, will take this. Thanks.